dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -1558,7 +1558,7 @@
     },
     "AnalyzeConversationLROTaskKind": {
       "type": "string",
-      "description": "Enumeration of supported analysis tasks on a collection of conversations.",
+      "description": "Enumeration of supported analysis tasks on a collection of conversation.",
```

Comment:
Actually, unless you were referring to a specific model, plural "conversations" is grammatically correct here.

Reply:
Resolved